[2022-07-28 09:52:43,818] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v4.s3_delete_objects manual__2022-07-28T09:51:01.397439+00:00 [queued]>
[2022-07-28 09:52:43,830] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v4.s3_delete_objects manual__2022-07-28T09:51:01.397439+00:00 [queued]>
[2022-07-28 09:52:43,831] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 09:52:43,831] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-28 09:52:43,831] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 09:52:43,853] {taskinstance.py:1397} INFO - Executing <Task(S3DeleteObjectsOperator): s3_delete_objects> on 2022-07-28 09:51:01.397439+00:00
[2022-07-28 09:52:43,861] {standard_task_runner.py:52} INFO - Started process 1569 to run task
[2022-07-28 09:52:43,868] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v4', 's3_delete_objects', 'manual__2022-07-28T09:51:01.397439+00:00', '--job-id', '215', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v4.py', '--cfg-path', '/tmp/tmpmtkrk8kl', '--error-file', '/tmp/tmpyd1zn_lh']
[2022-07-28 09:52:43,870] {standard_task_runner.py:80} INFO - Job 215: Subtask s3_delete_objects
[2022-07-28 09:52:44,028] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v4.s3_delete_objects manual__2022-07-28T09:51:01.397439+00:00 [running]> on host 2c2de32fb47f
[2022-07-28 09:52:44,214] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=abin404@yopmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v4
AIRFLOW_CTX_TASK_ID=s3_delete_objects
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T09:51:01.397439+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T09:51:01.397439+00:00
[2022-07-28 09:52:44,240] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-28 09:52:44,241] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-28 09:52:44,916] {s3.py:837} INFO - Deleted: ['yellow_tripdata_2020-04.csv', 'yellow_tripdata_2020-05.csv']
[2022-07-28 09:52:44,948] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v4, task_id=s3_delete_objects, execution_date=20220728T095101, start_date=20220728T095243, end_date=20220728T095244
[2022-07-28 09:52:45,006] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 09:52:45,093] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
